recipes/filesendercli/LICENSE.txt

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
It looks like all of the source files contain a copy of the license, so adding this is unnecessary. You can remove it and the line referencing it in the `meta.yaml`. Just add a comment saying it's in the source code in the `meta.yaml`. The linter will complain, but that's ok.